docs_app/content/tutorial/2-unsubscribing.md

```diff
@@ -0,0 +1,39 @@
+# 2. Unsubscribing
+
+## TLDR:
+
+- Calls to [`subscribe`](API) return a [`Subscription`](API) object.
+- Calling [`unsubscribe`](API) on the [`Subscription`](API) object notifies the [producer](GL) to stop sending values, and triggers the [teardown](GL) of any underlying resources the [producer](GL) has registered for teardown.
+- Failure to [unsubscribe](GL) from asynchronous observables will result in unnecessary resource use, and even memory leaks.
```

Comment:
Shouldn't that be balanced by saying that a completion or an error automatically unsubscribes? In my experience with Angular, most observables (HTTP, Form, ActivatedRoute observables) are short-lived and guaranteed to complete or error.